client/js/widgets/imagewidget.js

```diff
@@ -12,8 +12,11 @@ class ImageWidget extends Widget {
 
   applyDeltaToDOM(delta) {
     super.applyDeltaToDOM(delta);
-    if(delta.text !== undefined || delta.icon !== undefined)
-      setText(this.domElement, this.get('icon') ? '' : this.get('text'));
+    if(delta.text !== undefined || delta.icon !== undefined) {
+      if(this.get('type') !== 'holder') {
```

Comment:
Holder doesn't have `setText`, right? Maybe do `typeOf this.setText == 'function'`?

Reply:
AI says `if (typeof setText === 'function') ` would always be true because the function exists as a global function and not a method on the holder class.  This line was added to prevent the text from appearing twice (once in the position it would be for a basic widget).